packages/block-library/src/video/editor.scss

```diff
@@ -1,29 +1,6 @@
 // Provide special styling for the placeholder.
 // @todo this particular minimal style of placeholder could be componentized further.
 .wp-block-video.wp-block-video {
-	// Show Placeholder style on-select.
```

Comment:
This css isn't required as we are displaying illustration now on block selection only.

Reply:
Can you expand on why this is changing?

Reply:
Looks like it already been handled by https://github.com/WordPress/gutenberg/pull/64861. I'll sync the PR with trunk.